db.init.js

```diff
@@ -0,0 +1,162 @@
+const mongoose = require('mongoose');
+const models = require('./models');
+
+const { Types } = mongoose;
+const { ObjectId } = Types;
+const {
+  Category, Order, OrderStage, Item, Filter, Discount,
+} = models;
+
+const dbConn = process.argv[2];
```

Comment:
@IgorChaikin Its not best practice. If your application is working with one database instance, you should use environment variables to specify connection string. For example if your developing script for data migration between databases you could add args to specify source and destination, but here, since you are using only one database, lets use env variables

More details here https://12factor.net/ru/